Classes/Form/AbstractFormComponent.php

```diff
@@ -355,6 +356,15 @@ public function setLocalLanguageFileRelativePath($localLanguageFileRelativePath)
      */
     public function getLocalLanguageFileRelativePath()
     {
+        if (!$this->localLanguageFileRelativePath) {
```

Comment:
This method gets called extremely often (every time any label is built) so retrieving extension configuration at this point is not a good solution in terms of performance.